common/changes/@cadl-lang/versioning/versioning-OverhaulVersionDecorators_2023-01-13-00-44.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/versioning",
+      "comment": "Adds the @useDependency decorator. Deprecates the @versionedDependency decorator.",
+      "type": "none"
```

Comment:
I noticed that `rush change` has stopped asking us what the version bump is.  Was that on purpose?  Maybe because we've locked versions? /cc @nguerrera

Reply:
Yes, it's because we locked versions. I noticed this too. I think it's OK to let it do this. I think it only removes the designation in changelog, which makes sense if we're not using these to determine the bump.

Reply:
Checking to be sure that it doesn't cause things to be dropped from the changelog altogether.